sktime/markovprocess/markov_state_model.py

```diff
@@ -19,19 +19,18 @@
 # .. moduleauthor:: B. Trendelkamp-Schroer <benjamin DOT trendelkamp-schroer AT fu-berlin DOT de>
 
 import copy
+import typing
 from math import ceil
 
 import numpy as np
-import typing
-
 from pyemma.util.linalg import mdot
 
 from sktime.base import Model
-from sktime.markovprocess.sample import _ensure_dtraj_list
-from sktime.util import ensure_ndarray
 from sktime.markovprocess import Q_
 from sktime.markovprocess._dtraj_stats import TransitionCountModel
 from sktime.markovprocess.pcca import PCCA
+from sktime.markovprocess.sample import _ensure_dtraj_list
+from sktime.util import ensure_ndarray
```

Comment:
maybe we can lump all these `ensure_...` methods together into one file

Reply:
will do